pkg/utils/utils_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package utils_test
+
```

Comment:
The [TESTABLE](https://github.com/appc/cni/blob/master/test#L14) list must be updated.